src/main/webapp/app/core/websocket/websocket.service.ts

```diff
@@ -180,6 +182,7 @@ export class JhiWebsocketService implements OnDestroy {
     }
 
     unsubscribe(channel?: string) {
+        // TODO: Fix access of myListeners[undefined]
```

Comment:
Can someone help me with this fix access of `myListeners[undefined]`?

Reply:
i removed the optional. this caused only one problem in the app. i fixed that issue